botnim/vector_store/vector_store_openai.py

```diff
@@ -26,6 +27,10 @@ def get_or_create_vector_store(self, context, context_name, replace_context):
         self.init = True
         return ret
 
+    def reset_init(self):
```

Comment:
Why?

Reply:
When processing multiple contexts in a single session (like `botnim sync staging takanon --replace-context`): the actual use case required creating potentially multiple vector stores (one per context).

The first context would be processed successfully, and self.init would be set to True. When the second context was processed, if its vector store didn't exist or if the name had changed (due to changes in the get_index_name function), the assertion would fail.

Reply:
It seems more correct to remove the init field altogether in that case

Reply:
like this?